tests/input_files/poro_3D_psi_hex8_new_struc.dat

```diff
@@ -0,0 +1,961 @@
+ELEMENTS                        125
+NODES                           216
+DIM                             3
+MATERIALS                       4
+------------------------------------------------------PROBLEM TYPE
+PROBLEMTYPE                     Poroelasticity
+RESTART                         0
+SHAPEFCT                        Polynomial
+----------------------------------------------------DISCRETISATION
+NUMALEDIS                       0
+NUMFLUIDDIS                     0
+NUMSTRUCDIS                     1
+----------------------------------------------------------------IO
+FILESTEPS                       1000
+FLUID_STRESS                    No
+FLUID_WALL_SHEAR_STRESS         No
+OUTPUT_BIN                      Yes
+STRUCT_DISP                     Yes
+STRUCT_STRAIN                   No
+STRUCT_STRESS                   No
+STRUCT_SURFACTANT               No
+THERM_HEATFLUX                  None
+THERM_TEMPERATURE               No
+THERM_TEMPGRAD                  None
+-----------------------------------------------------FLUID DYNAMIC
+CALCERROR                       no
+CONVFORM                        convective
+GRIDVEL                         BE
+INITIALFIELD                    zero_field
+THETA                           0.66
+TIMEINTEGR                      One_Step_Theta
+RESULTSEVRY                           1
+LINEAR_SOLVER                   3
+---------------------------FLUID DYNAMIC/POROUS-FLOW STABILIZATION
+CROSS-STRESS                    no_cross
+GRAD_DIV                        no
+DEFINITION_TAU                  Franca_Madureira_Valentin_Badia_Codina
+EVALUATION_MAT                  integration_point
+EVALUATION_TAU                  integration_point
+PSPG                            yes
+REYNOLDS-STRESS                 no_reynolds
+RSTAB                           rstab_gls
+STABTYPE                        residual_based
+SUPG                            no
+TDS                             quasistatic
+TRANSIENT                       no_transient
+VSTAB                           no_vstab
+--------------------------------------------POROELASTICITY DYNAMIC
+COUPALGO                        poro_monolithic
+PHYSICAL_TYPE                   Poro
+ITEMAX                          20
+ITEMIN                          1
+MAXTIME                         1.0
+NORMCOMBI_RESFINC               And
+NORM_INC                        AbsSingleFields
+NORM_RESF                       AbsSingleFields
+NUMSTEP                         10
+RESTARTEVRY                     5
+TOLINC_DISP                     1e-06
+TOLINC_VEL                      1e-06
+TOLINC_PRES                     1e-06
+TOLINC_PORO                     1e-06
+TOLRES_DISP                     1e-04
+TOLRES_VEL                      1e-04
+TOLRES_PRES                     1e-04
+TOLRES_PORO                     1e-04
+VECTORNORM_RESF                 L1
+VECTORNORM_INC                  Rms
+TIMESTEP                        0.01
+RESULTSEVRY                           1
```

Comment:
This file itself is not super small and we are doing 100 computation steps, and writing output on every step.

Is this really necessary or might it be sufficient to

 * use a smaller test
 * larger timestep size / fewer timesteps
 * write output only on few points in time?

I haven't checked how long the test is running in the pipeline, but to me it looks like a little too complex and runtime could be reduced a lot with just a few modifications.

The same applies to the other 3D input file. Probably somehow related to #118.

Reply:
If not testing for output, there's for sure no need to write results in every time step.